docs/docs/api/endpoints/messages.md

```diff
@@ -81,6 +81,16 @@ to see learn how to send text, media, and many more message types.
 
 **Sample response**
 
+Because of the nature of being an asynchronous system, sometimes the conversation is not ready at the time of sending
+the message.
+
+`HTTP 200 OK`
+In this case the conversation was found and the message is attached to it
+
+`HTTP 202 ACCEPTED`
```

Comment:
If the conversation is ready there is no need to do this asynchronously. basically from the caller prospective `200` and `202` are basically the same call. the only thing that the caller need to be aware is that some other calls like `message.list` or `message.suggest-replies` won't reply with the expected data if the message is not ingested yet.

But for the propuse of the issue that is to be able to react instantly to a message it can be treated like a regular `200`